src/runners/rfqm_meta_transaction_consumer_runner.ts

```diff
@@ -0,0 +1,134 @@
+    PROMETHEUS_PORT,
+    RFQM_MAKER_ASSET_OFFERINGS,
+    RFQM_META_TX_SQS_URL,
+    RFQT_MAKER_ASSET_OFFERINGS,
+    SWAP_QUOTER_OPTS,
+} from '../config';
+import { METRICS_PATH, PROTOCOL_FEE_UTILS_POLLING_INTERVAL_IN_MS } from '../constants';
+import { getDBConnectionAsync } from '../db_connection';
+import { logger } from '../logger';
+import { RfqmService } from '../services/rfqm_service';
+import { providerUtils } from '../utils/provider_utils';
+import { RfqBlockchainUtils } from '../utils/rfq_blockchain_utils';
+import { SqsConsumer } from '../utils/sqs_consumer';
+
+import { getAxiosRequestConfig } from './http_rfqm_service_runner';
+
+const RFQM_JOB_DEQUEUED = new Counter({
+    name: 'rfqm_job_dequeued',
+    help: 'An Rfqm Job was pulled from the queue',
+});
+
+const RFQM_JOB_SUCCEEDED = new Counter({
+    name: 'rfqm_job_succeeded',
+    help: 'An Rfqm Job succeeded',
+});
+
+process.on('uncaughtException', (err) => {
+    logger.error(err);
+    process.exit(1);
+});
+
+process.on('unhandledRejection', (err) => {
+    if (err) {
+        logger.error(err);
+    }
+});
+
+if (require.main === module) {
+    (async () => {
+        // Start the Metrics Server
+        startMetricsServer();
+
+        // Build dependencies
+        const provider = providerUtils.createWeb3Provider(defaultHttpServiceWithRateLimiterConfig.ethereumRpcUrl);
+        const contractAddresses = await getContractAddressesForNetworkOrThrowAsync(provider, CHAIN_ID);
+        const quoteRequestor = new QuoteRequestor(
+            RFQT_MAKER_ASSET_OFFERINGS,
+            RFQM_MAKER_ASSET_OFFERINGS,
+            Axios.create(getAxiosRequestConfig()),
+            undefined, // No Alt RFQM offerings at the moment
+            logger.warn.bind(logger),
+            logger.info.bind(logger),
+            SWAP_QUOTER_OPTS.expiryBufferMs,
+        );
+        const protocolFeeUtils = ProtocolFeeUtils.getInstance(
+            PROTOCOL_FEE_UTILS_POLLING_INTERVAL_IN_MS,
+            ETH_GAS_STATION_API_URL,
+        );
+        const metaTxWorkerRegistry = META_TX_WORKER_REGISTRY || NULL_ADDRESS;
+        const exchangeProxy = new IZeroExContract(contractAddresses.exchangeProxy, provider);
+        const rfqBlockchainUtils = new RfqBlockchainUtils(exchangeProxy);
+        const connection = await getDBConnectionAsync();
+        const rfqmService = new RfqmService(
+            quoteRequestor,
+            protocolFeeUtils,
+            contractAddresses,
+            metaTxWorkerRegistry,
+            rfqBlockchainUtils,
+            connection,
+        );
+
+        // Run the consumer
+        await runRfqmMetaTransactionConsumerAsync(rfqmService);
+    })().catch((error) => logger.error(error.stack));
+}
+
+/**
+ * Runs the Rfqm Consumer
+ */
+export async function runRfqmMetaTransactionConsumerAsync(rfqmService: RfqmService): Promise<SqsConsumer> {
+    const consumer = new SqsConsumer({
+        sqs: new SQS({ apiVersion: '2012-11-05' }),
```

Comment:
What is this `apiVersion` about?

Reply:
just an AWS thing - to lock in the version of the API https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/SQS.html

I don't think the API is going to change at all, but just in case it does, we'll be fine